src/IOST/Signer.cpp

```diff
@@ -0,0 +1,106 @@
+
+#include "Signer.h"
+#include "../Hash.h"
+#include "../PrivateKey.h"
+#include <boost/endian/conversion.hpp>
+#include <sstream>
+
+#include <iostream>
+
+using namespace TW;
+using namespace TW::IOST;
+
+class IOSTEncoder {
+  public:
+    IOSTEncoder() {}
+    void WriteByte(uint8_t b) { buffer << b; }
```

Comment:
just curious we have `src/BinaryCoding.h`, could we reuse them?

Reply:
the encoding method is related to signature, so we have to implement it

Reply:
please take a look at `BinaryCoding::encode16BE`